Sources/SwifterSwift/SwiftStdlib/FloatingPointExtensions.swift

```diff
@@ -81,3 +81,15 @@ public prefix func ± <T: FloatingPoint> (number: T) -> (T, T) {
     return 0 ± number
 }
 // swiftlint:enable identifier_name
+
+// swiftlint:disable identifier_name
```

Comment:
I think that would be a good thing to add those exceptions on the swiftlint.yaml file ... common cases like those to avoid this bunch of enable disable ... no in this PR though, might be good for an issue with starter task :)